tests/components/zwave_js/test_api.py

```diff
@@ -113,6 +113,137 @@ async def test_node_status(hass, integration, multisensor_6, hass_ws_client):
     assert msg["error"]["code"] == ERR_NOT_LOADED
 
 
+async def test_node_metadata(hass, integration, wallmote_central_scene, hass_ws_client):
```

Comment:
It doesn't matter for this test, but normally we should load the device fixtures before loading the integration fixture to have the integration pick up the loaded devices.

This test doesn't test integration device or entity features but access the node directly, so it doesn't matter at the moment.

Reply:
ah interesting, I had no idea that order mattered. I will update it since it's the right thing to do

Reply:
Multiple tests had the order wrong so I updated all of them